service/discovery/azure/azure.go

```diff
@@ -304,16 +305,8 @@ func (d *azureDiscovery) discoverBackupVaults() error {
 				// Store voc.Backup in backupMap
 				d.backupMap[util.Deref(instance.Properties.DataSourceInfo.DatasourceType)][util.Deref(instance.Properties.DataSourceInfo.ResourceID)] = &voc.Backup{
 					Enabled:         true,
-					Policy:          util.Deref(instance.Properties.PolicyInfo.PolicyID),
 					RetentionPeriod: retentionDuration(util.Deref(retention)),
 					Storage:         voc.ResourceID(util.Deref(instance.ID)),
-					GeoLocation: voc.GeoLocation{
```

Comment:
Any particular reason for deleting this?

Reply:
The geolocation can also be found in the corresponding resource object.